data/bunrodea/bunrodea ships.txt

```diff
@@ -0,0 +1,499 @@
+	gun 5 -114 "Locust Blaster"
+	gun -14 -105.5 "Mandible Cannon"
+	gun 14 -105.5 "Mandible Cannon"
+	gun -24.5 -129 "Mandible Cannon"
+	gun 24.5 -129 "Mandible Cannon"
+	gun -54.5 -20 "Swarm Pod"
+	gun 54.5 -20 "Swarm Pod"
+	gun -62.5 -9.5 "Swarm Pod"
+	gun 62.5 -9.5 "Swarm Pod"
+	turret 0 54.5
+	leak "leak" 50 50
+	leak "flame" 80 80
+	leak "big leak" 80 30
+	explode "tiny explosion" 80
+	explode "small explosion" 40
+	explode "medium explosion" 40
+	explode "large explosion" 30
+	explode "huge explosion" 10
+	"final explode" "final explosion medium"
+	description "The Kama gunship is a formidable warship, capable of carrying up to ten guns as well as having a single turret mount typically equipped with an anti-missile turret, making the Kama less susceptible to missile weaponry than its smaller sibling, the Kunai."
+
+
+
+ship "Chigiriki"
+	sprite "ship/chigiriki"
+	thumbnail "thumbnail/chigiriki"
+	attributes
+		category "Heavy Warship"
+		"cost" 51460000
+		"shields" 116700
+		"hull" 46700
+		"required crew" 49
+		"bunks" 69
+		"mass" 650
+		"drag" 9.4
+		"heat dissipation" 0.7
+		"fuel capacity" 900
+		"cargo space" 102
+		"outfit space" 747
+		"weapon capacity" 504
+		"engine capacity" 163
+		weapon
+			"blast radius" 410
+			"shield damage" 4100
+			"hull damage" 2050
+			"hit force" 6600
+	outfits
+		"Thorax Cannon"
+		"Locust Blaster" 6
+		"Swarm Pod" 10
+		"Swarm Missile" 3000
+		
+		"Dark Reactor"
+		"Reactor Overclocker" 6
+		"Solar Battery" 3
+		"Large Shield Relay"
+		"Shield Relay Booster" 6
+		"Small Nanite Fabricator"
+		"Outfits Expansion" 2
+		
+		"Ookii Rift Thruster"
+		"Subarashii Rift Steering"
+		"Hyperdrive"
+		
+	engine -75 155
+	engine 75 155
+	engine -22.5 183.5
+	engine 22.5 183.5
+	gun 0 -143 "Thorax Cannon"
+	gun -14.5 -153 "Locust Blaster"
+	gun 14.5 -153 "Locust Blaster"
+	gun -24 -157.5 "Locust Blaster"
+	gun 24 -157.5 "Locust Blaster"
+	gun -33.5 -175 "Locust Blaster"
+	gun 33.5 -175 "Locust Blaster"
+	gun -67 -26.5 "Swarm Pod"
+	gun 67 -26.5 "Swarm Pod"
+	gun -75 12.5 "Swarm Pod"
+	gun 75 12.5 "Swarm Pod"
+	gun -82.5 21.5 "Swarm Pod"
+	gun 82.5 21.5 "Swarm Pod"
+	gun -95.5 85.5 "Swarm Pod"
+	gun 95.5 85.5 "Swarm Pod"
+	gun -106.5 105.5 "Swarm Pod"
+	gun 106.5 105.5 "Swarm Pod"
+	turret -20.5 40
+	turret 20.5 40
+	leak "leak" 30 50
+	leak "flame" 30 80
+	leak "big leak" 30 30
+	explode "tiny explosion" 90
+	explode "small explosion" 70
+	explode "medium explosion" 45
+	explode "large explosion" 20
+	explode "huge explosion" 10
+	"final explode" "final explosion large"
+	description "The Chigiriki is the largest of the Bunrodea's gunships, boasting an astonishing seventeen gun ports, one of which is equipped with the devastating Thorax Cannon, a massive canon capable of dealing significant damage to any ship that dares get in the Chigiriki's way. Some Chigiriki also double as missile platforms, raining down a walls of missiles so thick on their target that it can blot out the nearest star."
```

Comment:
```suggestion
	description "The Chigiriki is the largest of the Bunrodea's gunships, boasting an astonishing seventeen gun ports. One of these ports is equipped with the devastating Thorax Cannon, a massive cannon capable of dealing significant damage to any ship that dares get in the Chigiriki's way. Some Chigiriki also double as missile platforms, raining down a walls of missiles so thick on their target that it can blot out the nearest star."
```